app/build.gradle

```diff
@@ -65,7 +65,7 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    api project(':kin-ecosystem-sdk')
+    api project(':sdk')
```

Comment:
why API and not implementation

Reply:
doesn't really matter, didn't change it.